### ISSUE TITLE
I’m taking a break

### ISSUE THREAD

**Issue by kskeigrshi** [kskeigrshi/to-arch.sh#30]
I’m too lazy to write anything for now. I do not know when this will be updated.

**Comment by tildethinexxbisense** [comment]
is it still safe to use? @kskeigrshi

**Comment by kskeigrshi** [comment]
> is it still safe to use? @kskeigrshi

Yes why not, distros usually don’t change their internals that easily. However Manjarno isn’t the usual distro, they can’t be trusted.

Just test it out yourself.
